proto/storage/alert.proto

```diff
@@ -200,9 +221,12 @@ message ListAlert {
     // Represents an alert on a kubernetes resource (configmaps, secrets, etc.)
     // An alert cannot be on more than one entity (deployment, container image, resource, etc.)
     ResourceEntity resource = 11;
+    // Represents an alert on a node
+    // An alert cannot be on more than one entity (deployment, container image, resource, etc.)
+    NodeEntity node = 16;
```

Comment:
Reading the comment above the `oneof` suggests this should be `12` and that the next available tag should remain `16`.

Reply:
Ah yeah good spot; fixed.